src/operator/nn/deconvolution-inl.h

```diff
@@ -122,7 +122,7 @@ struct DeconvolutionParam : public dmlc::Parameter<DeconvolutionParam> {
     // Modified by Li.bs
     // Use tag to control the calculation of pad
     bool bCal = false;
-    if (target_shape.ndim() != 0) {
```

Comment:
`target_shape` is default to `mxnet::TShape(0)`, so the original code is correct.

Reply:
Correct. Revert this accordingly